src/big-trend/big-trend.js

```diff
@@ -348,7 +358,7 @@ export class BigTrend extends mixinBehaviors(
 			},
 			_trendItems: {
 				type: Array,
-				computed: '_getTrendItems(trendData)'
+				computed: '_getTrendItems(trendData, demonstrationProviderActivities)'

```

Comment:
This is unused so shouldn't be necessary
```suggestion
				computed: '_getTrendItems(trendData)'
```